docs/products/opensearch/reference/plugins.rst

```diff
@@ -21,17 +21,13 @@ These plugins are enabled on all Aiven for OpenSearch services by default:
 * Mapper Size
```

Comment:
Some plugins have links, others don't, why so and what's the difference?

Reply:
The ones without links didn't have any previously either, but I've added them in now ;)